### ISSUE TITLE
Confusing .import/ with *.import in .gitignore

### ISSUE THREAD

**Issue by Sslaxx** [rakugoteam/Rakugo-archive#128]
Rakugo's .gitignore file includes `*.import`, which is incorrect. It is also a sign of confusion with `.import/` the directory.

`*.import` files give Godot information about the associated asset files - for example, if they're textures if they repeat or not, loop offsets for sound files and so on. For your project to deal with asset files properly, it's *essential* these are included.

The `.import/` directory contains cached copies of the assets.

TL;DR - have `.import/` in .gitignore, but don't have `*.import`

**Comment by Jeremi360** [comment]
O my bad I will fix it.

**Comment by Jeremi360** [comment]
Fixed :)

**Comment by Sslaxx** [comment]
Thank you.

Fixed by https://github.com/jebedaia360/Rakugo/pull/129
